salt/returners/redis_return.py

```diff
@@ -184,17 +209,25 @@ def _get_serv(ret=None):
     if REDIS_POOL:
         return REDIS_POOL
     elif _options.get("cluster_mode"):
-        REDIS_POOL = StrictRedisCluster(
-            startup_nodes=_options.get("startup_nodes"),
-            skip_full_coverage_check=_options.get("skip_full_coverage_check"),
+        nodes = []
+        for node in _options.get("startup_nodes"):
```

Comment:
**suggestion**: I'm not positive that startup_nodes will *always* be an iterable instead of say, None, so it makes sense to provide a fallback value for `.get`

I would also just use a list comprehension rather than a for loop:

```
nodes = [ClusterNode(**node) for node in _options.get("startup_nodes", [])]
```
I think it's fine to assign to a variable, even though it *could* go into the `startup_nodes=[...]`. That form is a bit more awkward when debugging.